src/react-apps/applications/service-development/src/App.tsx

```diff
@@ -75,9 +77,12 @@ class App extends React.Component<IServiceDevelopmentProps, IServiceDevelopmentA
   }
 
   public componentDidMount() {
-    const altinnWindow: Window = window;
+    const altinnWindow: any = window;
```

Comment:
Why did you change type: Window to any?

Reply:
to make this work:  const { org, service } = altinnWindow;
That does not work with Window